tests/framework/common.go

```diff
@@ -105,7 +105,7 @@ var (
 	// default deploy Fluentbit
 	defaultDeployFluentbit = false
 	// default envoy loglevel
-	defaultEnvoyLogLevel = "debug"
+	defaultEnvoyLogLevel = "info"
```

Comment:
@eduser25 don't we want to go the exact opposite way?  Isn't it better to dump as many logs as possible during tests - so that WHEN test breaks we have context?

Reply:
was a test to see if it did affect testing anyhow. Envoy explicitely does not recommend touching logs.
We are reverting back anyway, it has not yielded any results.